Makefile

```diff
@@ -246,7 +246,10 @@ endif
 endif
 endif
 
-run: check_certs
+fetch_supermq:
+	@./scripts/supermq.sh
+
```

Comment:
Add this to CI, so it performs chek if there are changes in the SMQ compose is up to date.